packages/node_modules/@webex/test-users/src/index.js

```diff
@@ -156,7 +156,7 @@ export function createTestUser(options = {}) {
   const clientId = options.clientId || process.env.WEBEX_CLIENT_ID;
   const clientSecret = options.clientSecret || process.env.WEBEX_CLIENT_SECRET;
   const idbrokerUrl = options.idbrokerUrl || process.env.IDBROKER_BASE_URL;
-  const conversationServiceUrl = options.conversationServiceUrl || process.env.WEBEX_TEST_USERS_CONVERSATION_SERVICE_URL;
+  const cigServiceUrl = options.cigServiceUrl || process.env.WEBEX_TEST_USERS_CI_GATEWAY_SERVICE_URL;
```

Comment:
@adamweeks that means that `WEBEX_TEST_USERS_CONVERSATION_SERVICE_URL` would need to be added back at the end here, correct?